iteration/iteration.go

```diff
@@ -179,3 +185,25 @@ func (m *GormIterationRepository) CanStartIteration(ctx context.Context, i *Iter
 	}
 	return true, nil
 }
+
+// LoadDefault returns single Iteration which is default to given space
+func (m *GormIterationRepository) LoadDefault(ctx context.Context, spaceInstance space.Space) (*Iteration, error) {
+	defer goa.MeasureSince([]string{"goa", "db", "iteration", "loaddefault"}, time.Now())
+	var obj Iteration
+
+	tx := m.db.Where("space_id = ? and name = ? and path = ''", spaceInstance.ID, spaceInstance.Name).First(&obj)
```

Comment:
thanks, done in https://github.com/almighty/almighty-core/pull/990/commits/ad1a2c90572d86079a5205dbbe00f142fa785118